proto/axelar/nexus/v1beta1/events.proto

```diff
@@ -43,3 +43,8 @@ message RateLimitUpdated {
   google.protobuf.Duration window = 3
       [ (gogoproto.stdduration) = true, (gogoproto.nullable) = false ];
 }
+
+message NewGeneralMessage {
+  string id = 1 [ (gogoproto.customname) = "ID" ];
+  bytes payload_hash = 2;
```

Comment:
I think we should add the various metadata too, such as cross chain sender/recipient, message status etc.
`ExecuteMessage` only needs to be called in certain scenarios, such as EVM -> Cosmos. This event will be emitted for other flows too, and adding the metadata will be useful for axelarons to decide this.